pywbem/cim_http.py

```diff
@@ -590,17 +594,21 @@ def connect(self):
             # Connect using Python SSL module
             else:
                 # Setup the socket context
+
+                # Note: PROTOCOL_SSLv23 allows talking to servers with TLS but
+                # not with SSL. For details, see the table in
+                # https://docs.python.org/3/library/ssl.html#ssl.wrap_socket
                 # TODO ks 4/16: confirm that we cannot use the default_context()
                 # Selects the highest protocol version that both the
                 # client and server support (SSLV23)
                 ctx = SSL.SSLContext(SSL.PROTOCOL_SSLv23)
 
-                # TODO ks 4/16: Is there a use for the CERT_OPTIONAL mode
                 if self.cert_file:
                     ctx.load_cert(self.cert_file, keyfile=self.key_file)
                 if self.ca_certs:
-                    # CERT_REQUIRED validates server certificate:
-                    # against certificates in ca_certs
+                    # We need to use CERT_REQUIRED to get the server
```

Comment:
I would prefer replacing the phrase 'to get the' with to require the' since we do not consider security an option